app/Http/Controllers/StatementController.php

```diff
@@ -85,8 +86,19 @@ public function getStatement(Request $request, Validate $validate)
             $campStatement =  Statement::getLiveStatement($filter);
             if ($campStatement) {
                 $campStatement->go_live_time = date('m/d/Y, H:i:s A', $campStatement->go_live_time);
+                $rootUrl = str_replace("/public","",$request->root());
+                $rootUrl = str_replace("/index.php","",$rootUrl);
+                $WikiParser = new wikiParser;
+                $output = $WikiParser->parse($campStatement->value);
+                $finalStatement = str_replace("http://canonizer.com",$rootUrl,$output);
```

Comment:
If you use regex carefully, you can reduce the 4 lines into one. i.e. `$finalStatement`.